lona/response_parser.py

```diff
@@ -50,7 +50,7 @@ def render_response_dict(self, raw_response_dict, view_name):
             # check keys
             if len(set(raw_response_dict) & key_words) != 1:
                 raise RuntimeError(
-                    'response dicts have to contain exactly one keyword of: {}'.format(  # NOQA: LN001
+                    'response dicts have to contain exactly one keyword of: {}'.format(  # NOQA: LN001,FS002
```

Comment:
Why do we leave this line like this?

Reply:
While migrating to f-strings in #58 I decided that in some cases `.format` is more readable.
All such cases now have `# NOQA`.
If you think it's better to rewrite them using f-strings, I can update PR.